bin/builder-core.js

```diff
@@ -38,6 +45,11 @@ module.exports = function (opts, callback) {
     runner: runner
   });
 
+  // Drain messages if any.
+  (opts.msgs || []).forEach(function (obj) {
```

Comment:
I've added a longer commentary of how we really have two different message queues -- https://github.com/FormidableLabs/builder/commit/6e9b658b1ee347b9869d325a45185cd340cca540